### DISCUSSION TITLE
怎样仅在播放时置顶

### DISCUSSION THREAD

**Discussion by Dovelong0713** [hooke007/mpv_PlayKit#453] [mpv 通用]
看了很多文章,置顶都是始终置顶,怎样仅在播放时置顶

**Comment by hooke007** [comment]
https://github.com/hooke007/MPV_lazy/blob/main/portable_config%2Fprofiles.conf#L152-L152

**Reply by Dovelong0713** [reply]
是我愚钝了，
我在\portable_config文件夹添加了你的profiles.conf文件，
在mpv.conf文件添加了profile=\<profiles\> ， 但是没有效果啊，哪里不对吗

**Reply by hooke007** [reply]
照抄当然没用，你都没看前置说明

**Comment by hooke007** [comment]
照抄用这个，放在mpv.conf里正确的位置，移除掉你前面涉及到重复的参数
```
[ontop_playback]
 profile-desc    = 播放时置顶
 profile-cond    = not pause
 profile-restore = copy
 ontop           = yes
```
